testsuite/tests/typing-layouts-or-null/stable.ml

```diff
@@ -0,0 +1,14 @@
+(* TEST
+ flags = "-extension-universe stable";
+ expect;
+*)
+
+(* CR dkalinichenko: [or_null] is not available in [stable]. *)
+let x : int or_null = This 3
```

Comment:
Let's also have tests to make sure `This` and `Null` are not available

Reply:
Done.